docs/admin/workloads/inference-overview.md

```diff
@@ -1,53 +1,52 @@
-
 ---
-title: Inference  Overview 
+title: Inference overview
+summary: This article describes inference worloads.
+authors:
+    - Jason Novich
+date: 2024-Mar-29
 ---
+
 ## What is Inference
 
-Machine learning (ML) inference is the process of running live data points into a machine-learning algorithm to calculate an output. 
+Machine learning (ML) inference is the process of running live data points into a machine-learning algorithm to calculate an output.
 
-With Inference, you are taking a trained _Model_ and deploying it into a production environment. The deployment must align with the organization's production standards such as average and 95% response time as well as up-time. 
+With Inference, you are taking a trained *Model* and deploying it into a production environment. The deployment must align with the organization's production standards such as average and 95% response time as well as up-time.
 
 ## Inference and GPUs
- 
-The Inference process is a subset of the original Training algorithm on a single datum (e.g. one sentence or one image), or a small batch. As such, GPU memory requirements are typically smaller than a full-blown Training process. 
 
-Given that, Inference lends itself nicely to the usage of Run:ai Fractions. You can, for example, run 4 instances of an Inference server on a single GPU, each employing a fourth of the memory. 
+The Inference process is a subset of the original Training algorithm on a single datum (e.g. one sentence or one image), or a small batch. As such, GPU memory requirements are typically smaller than a full-blown Training process.
+
+Given that, Inference lends itself nicely to the usage of Run:ai Fractions. You can, for example, run 4 instances of an Inference server on a single GPU, each employing a fourth of the memory.
 
 ## Inference @Run:ai
 
-Run:ai provides Inference services as an equal part together with the other two Workload types: _Train_ and _Build_.
+Run:ai provides Inference services as an equal part together with the other two Workload types: *Train* and *Build*.
```

Comment:
do we still call it "build"? (rather than interactive/workspace?) just making sure it's consistent with the other docs

Reply:
@shayasoolin I am not sure what this sentence is supposed to mean. I can make it more generic.

```suggestion
Run:ai provides Inference services as an equal part together with any other Workload type that is available.
```